build/yaml/crd/vpc/crd.nsx.vmware.com_subnets.yaml

```diff
@@ -89,6 +92,15 @@ spec:
                 - message: Value is immutable
                   rule: self == oldSelf
             type: object
+            x-kubernetes-validations:
+            - message: DHCPConfig is required once set
+              rule: '!has(oldSelf.DHCPConfig) || has(self.DHCPConfig)'
+            - message: ipv4SubnetSize is required once set
+              rule: '!has(oldSelf.ipv4SubnetSize) || has(self.ipv4SubnetSize)'
+            - message: accessMode is required once set
```

Comment:
since we also changed accessMode and ipv4subnetsize to be required once set, it;s good that we also have a test and update the commits msg accordingly. Thanks.

Reply:
Updated, thanks!